ui/app/components/app/network-display/network-display.js

```diff
@@ -45,7 +52,9 @@ export default function NetworkDisplay({
             size={indicatorSize}
             type={ColorIndicator.TYPES.FILLED}
             iconClassName={
-              networkType === NETWORK_TYPE_RPC ? 'fa fa-question' : undefined
+              networkType === NETWORK_TYPE_RPC && indicatorSize !== SIZES.XS
```

Comment:
Why does the indicator size affect whether or not `fa-question` is used as the icon? Is there no question icon at the extra small size?

Reply:
It looks pretty bad at this size, scrunched and distorted

Reply:
Also the design does not have one